test/cases/grid-template.css

```diff
@@ -69,10 +69,26 @@
 @media (min-width: 1000px) {
     .b {
         grid-template:
-            [header-left] "head" 30px [header-right]
-            [nav-left]    "nav"  auto  [nav-right]
-            [main-left]   "main" 1fr  [main-right]
-            [footer-left] "foot" 30px [footer-right]
+            [header-left] 'head' 30px [header-right]
+            [nav-left] 'nav' auto [nav-right]
+            [main-left] 'main' 1fr [main-right]
+            [footer-left] 'foot' 30px [footer-right]
             / 1fr;
     }
 }
+
+.foo {
```

Comment:
I think it's better to change existing tests than adding new after all. We already have a large number of tests. And the further the more difficult it becomes to understand them.

Reply:
I like more tests 😄

Reply:
And I like more test cases :) In this situation, we duplicate the same test cases many times. We don't increase coverage, just increase the amount of code.

Reply:
Good point 👍

Reply:
I suggest leaving only `.az` and `.aa`